hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/HFileInputFormat.java

```diff
@@ -109,6 +114,9 @@ public boolean nextKeyValue() throws IOException, InterruptedException {
         return false;
       }
       value = scanner.getCell();
+      if (value != null && bulkloadSeqId.isPresent()) {
+        PrivateCellUtil.setSequenceId(value, bulkloadSeqId.getAsLong());
```

Comment:
We can not use bulkloadSeqId.ifPresent because PrivateCellUtil.setSequenceId will throw exceptions?

Reply:
Yea setSequenceId throws IOException. Figured it looks cleaner this way